rpcs3/rpcs3qt/breakpoint_list.cpp

```diff
@@ -78,16 +84,36 @@ void breakpoint_list::AddBreakpoint(u32 pc)
 	addItem(breakpoint_item);
 
 	Q_EMIT RequestShowAddress(pc);
+
+	return true;
 }
 
 /**
 * If breakpoint exists, we remove it, else add new one.  Yeah, it'd be nicer from a code logic to have it be set/reset.  But, that logic has to happen somewhere anyhow.
 */
 void breakpoint_list::HandleBreakpointRequest(u32 loc)
 {
-	if (!m_cpu || m_cpu->id_type() != 1 || !vm::check_addr(loc, vm::page_allocated | vm::page_executable))
+	if (!m_cpu || m_cpu->state & cpu_flag::exit)
+	{
+		return;
+	}
+
+	if (m_cpu->id_type() != 1)
 	{
 		// TODO: SPU breakpoints
+		QMessageBox::warning(this, tr("Unimplemented Breakpoints For Thread Type!"), QObject::tr("Cannot set breakpoints on non-PPU thread currently, sorry."));
```

Comment:
Please remove all the static QObject::tr calls. Use the regular ones unless the class doesn't inherit it.
Otherwise the string won't show in this class in the translator

Reply:
This

Reply:
Done.